src/main/java/org/getopt/luke/plugins/FsDirectory.java

```diff
@@ -157,6 +158,11 @@ public void renameFile(String from, String to) throws IOException {
     fs.rename(new Path(directory, from), target);
   }
 
+  @Override
+  public void syncMetaData() throws IOException {
+
```

Comment:
@jpbempel ok, neither do I. Well, we will let hadoop folks to revisit this, if there are any that use luke atop hdfs.